### ISSUE TITLE
Possible to configure verbose output after launch?

### ISSUE THREAD

**Issue by stemcc** [Johni0702/mumble-web-proxy#12]
I'm testing out mumble-web-proxy and have found that no terminal / stdout updates after the proxy has launched is making troubleshooting rather difficult. Right now the proxy exits within 20 seconds of launch, and not having any feedback means I'm sort of feeling around in the dark trying to isolate the issue. Are there logs you can point me to that the proxy might be writing?

**Comment by Johni0702** [comment]
It logs to stdout/stderr. Are you saying that it doesn't do that for you or that you do not have access to stdout/stderr?

**Comment by streaps** [comment]
It doesn't log anything on startup though. The first log output I see is on a new connection:
 ```
New connection from [::1]:37729
```

**Comment by stemcc** [comment]
> It logs to stdout/stderr. Are you saying that it doesn't do that for you or that you do not have access to stdout/stderr?

I do have access to stdout/stderr. My guess is that my issues were related to the IP address I was using for the mumble server, as once I changed the mumble server in the mumble-proxy command to be the VM's IP address instead of the IP address resolved by the domain, I was able to make my first handshake with mumble-web webrtc client, and the logs to stdout/stderr were visibile (new connections, ice connection info, etc).

Before that, however, while I was still troubleshooting getting it to run, mumble-web-proxy would still launch, but there was no output to stdout/stderr to indicate the proxy wasn't happy with the mumble server as I had it. Since this was my first time using the proxy, I was unsure whether or not it had launched correctly, and was additionally unsure if there was logging because it never made it to that point. 

I think it might be worthwhile to consider even a little bit of logging before the first connection has been made to the proxy to help others when setting this up for the first time. Even something as simple as "Initialized . . . waiting for Murmur" might be really useful.
